doc/content/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Home"
```

Comment:
And then this is `Overview` of what to be found here

Reply:
Renamed